reference/src/client/test/client_test_helpers.h

```diff
@@ -400,12 +400,19 @@ namespace client_test_helpers {
      * @param[out] out output buffer.
      * @param[in] in input buffer.
      * @param[in] evp_pkey RSA key.
+     * @param[in] digest_algorithm the digest algorithm for OAEP padding.
+     * @param[in] mgf1_digest_algorithm the digest algorithm for the MGF1 function.
+     * @param[in] label the label for the OAEP padding. May be NULL.
+     * @param[in] label_length the length of the label. Must be 0 if label is NULL.
```

Comment:
Is this extraneous or needed in the function prototype?

Reply:
Extraneous. Done